assets/scss/_button.scss

```diff
@@ -1,10 +1,12 @@
+@import 'variables';
+
 .d-button {
-  @extend .button;
+  @extend .button !optional;
```

Comment:
what does `!optional` do?

Reply:
this was a workaround that i remove in a later commit

Reply:
is the latest pushed to github? I still see the `!optional` in the diff